lib/sidekiq/cron/web_extension.rb

```diff
@@ -56,24 +56,24 @@ def self.registered(app)
 
         # Enqueue cron job.
         app.post '/cron/namespaces/:namespace/jobs/:name/enque' do
-          if job = Sidekiq::Cron::Job.find(route_params[:name], route_params[:namespace])
+          if (job = Sidekiq::Cron::Job.find(route_params[:name], route_params[:namespace]))
```

Comment:
Is this change necessary? I'd prefer to avoid unnecessary diff 🙏

Reply:
Not necessary, just following best practice guidelines, to add clarifying parentheses.